core/test/utils/file_utils_test.dart

```diff
@@ -38,6 +41,60 @@ void main() {
 
       file.delete();
     });
+
+    test(
+      'Should delete file '
+      'when deleteCompressedFileOnMobile is called '
+      'and file exists '
+      'and Platform is mobile',
+    () async {
+      // arrange
+      final file = await FileUtils().saveToFile(nameFile: fileName, content: fileContent);
+
+      // act
+      await FileUtils().deleteCompressedFileOnMobile(file.path, pathContains: fileName);
+
+      // assert
+      expect(await file.exists(), equals(false));
+    });
+
+    test(
+      'Should not delete file '
+      'when deleteCompressedFileOnMobile is called '
+      'and file exists '
+      'and Platform is not mobile',
+    () async {
+      // arrange
+      debugDefaultTargetPlatformOverride = TargetPlatform.macOS;
```

Comment:
why debugDefaultTargetPlatformOverride = TargetPlatform.macOS? IMO, it should be TargetPlatform.Android or TargetPlatform.IOS.

Reply:
The test's description explicitly said `Platform is not mobile`